Makefile.am

```diff
@@ -86,7 +88,7 @@ $(OSX_APP)/Contents/PkgInfo:
 
 $(OSX_APP)/Contents/Resources/empty.lproj:
 	$(MKDIR_P) $(@D)
-	@touch $@ 
+	@touch $@
```

Comment:
Why the unrelated whitespace changes?

Reply:
My editor is set to pedantic mode. Not sure why we missed this in #13753.